packages/react-ui-kit/src/Icon/Files/TextFileIcon.tsx

```diff
@@ -21,9 +21,9 @@ import {SVGIcon, SVGIconProps} from '../SVGIcon';
 
 export const TextFileIcon = ({...props}: SVGIconProps) => {
   return (
-    <SVGIcon realWidth={14} realHeight={16} fill="none" {...props}>
+    <SVGIcon realWidth={13} realHeight={16} fill="none" {...props}>
       <path
-        d="M0.5 3.5C0.5 2.39543 1.39543 1.5 2.5 1.5H10.5C11.6046 1.5 12.5 2.39543 12.5 3.5V11.5063C12.5 12.1626 12.2419 12.7926 11.7815 13.2603L10.3108 14.754C9.84094 15.2312 9.19916 15.5 8.52938 15.5H2.5C1.39543 15.5 0.5 14.6046 0.5 13.5V3.5Z"
+        d="M2.5 1.5H10.5C11.6046 1.5 12.5 2.39543 12.5 3.5V11.5059C12.5 12.0801 12.3021 12.6343 11.9443 13.0771L11.7812 13.2607L10.3105 14.7539C9.84065 15.2311 9.19902 15.5 8.5293 15.5H2.5C1.39543 15.5 0.5 14.6046 0.5 13.5V3.5C0.5 2.39543 1.39543 1.5 2.5 1.5Z"
```

Comment:
Just making sure, this icon is intended to be tweaked, it's not a mistake?

Reply:
Yes, it's intended :)